src/main/java/fi/tkgwf/ruuvi/Main.java

```diff
@@ -119,7 +123,9 @@ boolean run(final BufferedReader reader) {
             }
         } catch (IOException ex) {
             LOG.error("Uncaught exception while reading measurements", ex);
-            return false;
+            if (Config.exitOnInfluxDBIOException()) {
```

Comment:
Looks like you added this check around the wrong return statement, should be in the InfluxDBIOException catch block

Reply:
Oops. :flushed: Fixed now